docs/account-update-contract.md

```diff
@@ -103,6 +103,13 @@ pub contract HelloWorld {
 
 Specify the name of the account that will be used to sign the transaction.
 
+### Show Diff
+
+- Flag: `--show-diff`
+- Valid inputs: `true`, `false`
```

Comment:
This might be minor, you don't need to specify true or false with flags. Maybe state the default value like other docs?